CONFIGURATION.md

```diff
@@ -105,7 +105,7 @@ Furthermore, if at all possible, rebooting should be limited to only those exit
 
 ## Systemd configuration
 
-To run NTPD-rs as the system ntp service, the following systemd service definition can be used. Note that this service definition assumes that the ntp-daemon binary has bin installed to /usr/local/bin, and that the configuration is stored in the default /etc/ntp.toml location.
+To run NTPD-rs as the system ntp service, the following systemd service definition can be used. Note that this service definition assumes that the ntp-daemon binary has bin installed to /usr/local/bin, and that the configuration is stored in the default /etc/ntp.toml location. Furthermore, it assumes the existence of a low-privileged `ntpd-rs` group and user.
```

Comment:
this should document how to create such a group/user

Reply:
I don't think this is the right place to document that, as it is both a fairly standard system administrative task, and the procedure for doing so is not entirely uniform across the Linux ecosystem, depending on which utilities are included in the distribution. I have added a short sentence referring the user to their distributions documentation on this.